### PR TITLE
[DOC-7242] Lambda env var

### LINKED ISSUE #1158: Issue: Enable serverless monitoring using the Lambda layer
Would be helpful that the documentation highlights all the available environment-variables for configuring and controlling module behavior.  While the helper-cli might be used by some clients; most enterprises will have their own CI/CD process and explicit definition of available controls would be helpful.

**Required Config:**
NEW_RELIC_LICENSE_KEY  ||  NEW_RELIC_LICENSE_KEY_SECRET
NEW_RELIC_LAMBDA_HANDLER
NEW_RELIC_ACCOUNT_ID

**Optional Config:**
NEW_RELIC_LAMBDA_EXTENSION_ENABLED            (defaults true, false activates legacy cloudwatch-log-output)
NEW_RELIC_EXTENSION_SEND_FUNCTION_LOGS   (defaults false)
NEW_RELIC_EXTENSION_LOG_LEVEL                          (defaults INFO)
NEW_RELIC_DISTRIBUTED_TRACING_ENABLED         (defaults false)

**Open Question:  Are these environment variables still required in the deployment of DT within Serverless?**
NEW_RELIC_PRIMARY_APPLICATION_ID
NEW_RELIC_TRUSTED_ACCOUNT_KEY

### Doc information (don't delete this section)

* https://docs.newrelic.com/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-serverless-monitoring-using-lambda-layer
* **OS**: Mac OS 11.2.2
* **Browser**: Chrome 88.0.4324.192
* **Device**: Unknown

### PR DESCRIPTION
**Pull request by urbiz-nr**
- Added environment var in Lambda docs
- Minor fixes in headers

See DOC-7242 for more details.

Fixes #1158.

### CODE REVIEW

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx [hunk 1]
```diff
@@ -170,10 +171,140 @@ com/secrets-manager/) for greater security.
...
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `NEW_RELIC_EXTENSION_LOG_LEVEL`
+          </td>
+
+          <td>
+            Used for debugging.
```
Comment (MattWhelan) [inline src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx:269]: "Set to "DEBUG" to enable debug logging.

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx [hunk 2]
```diff
@@ -170,10 +171,140 @@ com/secrets-manager/) for greater security.
...
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `NEW_RELIC_LAMBDA_EXTENSION_ENABLED`
+          </td>
+
+          <td>
+            TBC
```
Comment (MattWhelan) [inline src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx:289]: Set to `FALSE` to disable the extension and use the cloudwatch telemetry path.

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx [hunk 3]
```diff
@@ -170,10 +171,140 @@ com/secrets-manager/) for greater security.
...
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `NEW_RELIC_LICENSE_KEY` / `NEW_RELIC_LICENSE_KEY_SECRET`
+          </td>
+
+          <td>
+            Unless stated otherwise, we use a reasonable default value for the license key secret.
```
Comment (MattWhelan) [inline src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx:299]: If you're going to document these, you might as well actually document them. the LK is the classic NR license key. It's the highest-priority source for the LK. It's not set by default.

The secret is the name of the AWS Managed Secret containing the license key. Assuming the LK var itself is not set, we look the key up there. We have a default for this, which integrates with the CLI tools. 

NB: it's really important that the contents of the LK secret be formatted correctly. We suggest using the CLI to set the secret up, as it does the right thing.

**Review by MattWhelan — CHANGES_REQUESTED** [review]

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx (hunk 3 above)
Comment (urbiz-nr) [reply src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx:299]: Thanks @MattWhelan - I was using the comments in the spreadsheet, which were very confusing, that's why I wanted to check with you. I'll rephrase this, could you have another look?

File: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx [hunk 4]
```diff
@@ -170,10 +171,142 @@ com/secrets-manager/) for greater security.
...
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `NEW_RELIC_LICENSE_KEY` / `NEW_RELIC_LICENSE_KEY_SECRET`
+          </td>
+
+          <td>
+            The license key is associated to your New Relic account, while the secret is the name of the AWS Managed Secret containing the license key. If `NEW_RELIC_LICENSE_KEY` is not set, we look for the key in `NEW_RELIC_LICENSE_KEY_SECRET`. To ensure the contents of the keys are formatted correctly, we recommend to use the CLI (#rec) to enable lambda monitoring.
```
Comment (urbiz-nr) [inline src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx:299]: @MattWhelan - is this accurate? Thanks!

**Review by MattWhelan — APPROVED** [review]

**Pull request merged**